frontend/src/components/core/MainMenu/MainMenu.scss

```diff
@@ -40,7 +40,7 @@
 
   /* The menu itself. */
   div.dropdown-menu {
-    width: 12rem;
+    width: 13rem;
```

Comment:
Why change this?

Reply:
Because the new "Cancel screencast" option needs a little bit more space